articles/azure-app-configuration/howto-import-export-data.md

```diff
@@ -479,6 +479,7 @@ From the Azure CLI, follow the steps below. If you don't have the Azure CLI inst
     | `--destination`        | Enter `appservice` to indicate that you're exporting data to App Service.                                                                                          | `appservice`                                                                                                              |
     | `--appservice-account` | Enter the App Service's ARM ID or use the name of the App Service, if it's in the same subscription and resource group as the App Configuration.                   | `/subscriptions/123/resourceGroups/my-as-resource-group/providers/Microsoft.Web/sites/my-app-service` or `my-app-service` |
     | `--label`              | Enter a label to export keys and feature flags with this label. If you don't specify a label, by default, you will only export keys and feature flags with no label. You can enter one label, enter several labels by separating them with `,`, or use `*` to take all of the labels in account. | `prod`                                                                                                                    |
+    | `--export-as-reference` `-r` | Optional. Flag to specify whether key-values are exported to App Service as App Configuration references. This exports references to the latest revision of key-value(s) in App Configuration. [Learn more](../app-service/app-service-configuration-references.md). |
 
     > [!IMPORTANT]
     > If the keys you want to export have labels, you must use the command `--label` and enter the corresponding labels. If you don't select a label, only keys without labels will be exported. Use a comma sign (`,`) to select several labels or use `*` to include all labels, including the null label (no label).
```

Comment:
This note is wrong. If there is anything we need to explain about the label, we should include it in the "--label" line anyway. Can you please delete this?